temporal-spring-boot-autoconfigure/src/main/java/io/temporal/spring/boot/autoconfigure/template/WorkerFactoryOptionsTemplate.java

```diff
@@ -58,12 +65,15 @@ public WorkerFactoryOptions createWorkerFactoryOptions() {
           .ifPresent(options::setUsingVirtualWorkflowThreads);
     }
 
+    List<WorkerInterceptor> interceptors =
+        new ArrayList<>(workerInterceptors != null ? workerInterceptors : Collections.EMPTY_LIST);
     if (tracer != null) {
       OpenTracingWorkerInterceptor openTracingClientInterceptor =
           new OpenTracingWorkerInterceptor(
               OpenTracingOptions.newBuilder().setTracer(tracer).build());
-      options.setWorkerInterceptors(openTracingClientInterceptor);
+      interceptors.add(openTracingClientInterceptor);
```

Comment:
@cretz @Sushisource Any thoughts on the default order here? I am putting the tracing interceptor at the end of the chain. unless we want to trace the interceptors themselves maybe?

Reply:
Interesting. In other SDKs we have them create the tracing interceptor instead of just the tracer, so it is their responsibility to choose order.

Hard to say whether at the end (is that innermost?) is a good default. Some users swallow or mutate errors, and the span may be better off reflecting the results of other people's interceptors.

Say I didn't want it at the end of the chain as a user, how hard would it be for me to provide a null `tracer` to this constructor here and put it at the beginning of the `workerInterceptors` list? Just curious, because both defaults have tradeoffs, so just wondering if there was a workaround.

Reply:
Yeah end would be innermost, to work around a user would provide a customizer to change the interceptors

Reply:
I can check what Quarkus did

Reply:
:+1: If there's a workaround (e.g. don't inject tracer, make the interceptor in the list yourself), that'd be ideal. I do think the tracing intercepter being the outermost makes sense to have the span cover what other interceptors are doing (from a time or exception POV).